components/TutorielRaccourci.tsx

```diff
@@ -24,7 +24,7 @@ export function TutorielRaccourci() {
           aria-label='Plus'
           className='inline w-6 h-6'
         />
-        puis &quot;Installer l&apos;application&quot;
+        puis &quot;Ajouter à l&apos;écran d&apos;accueil&quot;
```

Comment:
```suggestion
        Ajouter à l’écran d’accueil
```
nitpick, mettre les vraies apostrophes, jpp de ces apos XD